ransack.gemspec

```diff
@@ -14,12 +14,13 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 1.9'
   s.license     = 'MIT'
 
-  s.add_dependency 'actionpack', '>= 5.0'
+  
   s.add_dependency 'activerecord', '>= 5.0'
   s.add_dependency 'activesupport', '>= 5.0'
   s.add_dependency 'i18n'
   s.add_dependency 'polyamorous', Ransack::VERSION.to_s
-
+  s.add_development_dependency 'actionpack', '>= 5.0'
```

Comment:
Development dependencies were moved to the `Gemfile` in https://github.com/activerecord-hackery/ransack/commit/8c92a3b1f6f2360599732f8d1ff4ae95f515eb86, and `actionpack` is already present in the current Gemfile, so you can remove this addition.